app/models.py

```diff
@@ -2,6 +2,17 @@
 from sqlalchemy.dialects.postgresql import JSON
 
 
+class Frameworks(db.Model):
+    __tablename__ = 'frameworks'
+
+    id = db.Column(db.Integer, primary_key=True)
+
+    name = db.Column(db.String(255), nullable=False)
+
+    expired = db.Column(db.Boolean, index=False, unique=False,
```

Comment:
Thought about dates, then I thought that when things expire is a movable issue, and keeping it up to date is an issue, so maybe a button and a boolean would be easier - we launch when we're ready etc rather than automatically. that might be one for the future.